src/server/pfs/server/source.go

```diff
@@ -20,6 +20,7 @@ type Source interface {
 }
 
 type source struct {
+	storage    *fileset.Storage
```

Comment:
Can we not depend on the whole fileset.Storage?  We seem to just grab the ChunkStorage() below.  I would prefer to pass that in in the constructor.

Reply:
If you agree with the change above, we would want to keep this.